doc/develop/test/sparse.rst

```diff
@@ -3,7 +3,8 @@
 Sparse support
 ##############
 
-`Sparse <https://kernelnewbies.org/Sparse>`__ is a static code analysis tool.
+`Sparse <https://www.kernel.org/doc/html/latest/dev-tools/sparse.html>`__
```

Comment:
single `_` should be sufficient

Reply:
the problem is that Sphinx complains about duplicate target (see L1 in this file).